src/java.base/share/classes/java/lang/Class.java

```diff
@@ -185,6 +189,32 @@
  * a class or interface is hidden has no bearing on the characteristics
  * exposed by the methods of class {@code Class}.
  *
+ * <h2><a id=unnamedClasses>Unnamed Classes</a></h2>
+ *
+ * A {@code class} file representing an {@linkplain #isUnnamedClass unnamed class} is generated by
```

Comment:
Are you going to reflow this paragraph before you are done, only wondering as there is one line sticking out compared the rest of the paragraph.

Reply:
Changing in primary PR - https://github.com/openjdk/jdk/pull/13689